bin/GameIndex.dbf

```diff
@@ -16652,12 +16658,6 @@ Region = PAL-M5
 ---------------------------------------------
 Serial = SLES-54466
 Name   = Test Drive Unlimited
-Region = PAL-M4
-Compat = 5
-VIFFIFOHack = 1 // Needed to load the main game properly.
----------------------------------------------
-Serial = SLES-53753
-Name   = Test Drive Unlimited
 Region = PAL-M5
```

Comment:
GitHub should've made this an (green) addition.
![image](https://user-images.githubusercontent.com/24227051/88109577-0f9cc580-cbab-11ea-9a47-31be96602442.png)

Not sure why it's doing that.

Reply:
Looks fine now for some reason.